tests/test_run.py

```diff
@@ -90,6 +90,7 @@
 basic_pheval_config = ExomiserConfig(
     run=ExomiserConfigRun(
         environment="local",
+        phenotype_only=False,
         path_to_exomiser_software_directory=Path(
             "/Users/yaseminbridges/exomiser/exomiser-cli-13.1.0"
```

Comment:
probably don't want this like this. `/opt/exomiser/exomiser-cli-13.1.0` or whatever.